Exercise_B/index.html

```diff
@@ -19,16 +19,118 @@
+            <!-- Payments Div -->
+          <div class="col-sm-2 mt-5 ms-3">
+            <p class = "">Flowers for every kind of situation</p>
+          </div>
+
+          <div class="col-sm-2">
+            <img src="./assets/images/payment-icon.png" alt="flowerShopLogo">
+
+            </div>
+          
+              <!-- Payments Div -->
+            <div class="col-sm-2 pt-4 ms-3">
+              <p class=" ">All major payment kinds accepted</p>
+            </div>
+            
+        </div>
+      </div>
+      
+      <div class ="container-fluid mt-5 #e6ffe6">
+        <div class="row justify-content-md-center">
+          <div class="col-sm-2">
+            <img src="./assets/images/phone-icon.png" alt="flowerShopLogo">
+          </div>
+        
+          <div class="col-sm-2 mt-5 ms-3">
+            <p>Free Consultation</p>
+          </div>
+
+          <div class="col-sm-2">
+            <!-- <figure> -->
+            <img src="./assets/images/truck-icon.png" alt="flowerShopLogo">
+          <!-- <figcaption class="">ff</figcaption> -->
+          </figure>
+            </div>
+          
+              <!-- Payments Div -->
+            <div class="col-sm-2 mt-5 ms-3">
+              <p class="">City wide delivery</p>
+            </div>
+            
+        </div>
+      </div>
+      
+      <div class ="container-fluid">
+        <div class="card justify-content-md-center" >
+          
+          <div class="card-body">
+            <h5 class="card-title">Testimonials</h5>
+            <p class="card-text">Hear what our customers have to say!</p>
+            <a href="#" class="btn btn-primary">Click here</a>
+          </div>
+        </div>
+
+          <div class="card justify-content-md-center">
+            <div class="card-body">
+            <h5 class="card-title">Pictures</h5>
+            <p class="card-text">More beautiful flowers await!</p>
+            <a href="#" class="btn btn-primary">Click here</a>
+          </div>
+        </div>
+      </div>
+      <!-- <div class="d-flex justify-content-center"> 
+        <div class="col-sm-2 ms-5">
+          <img src="./assets/images/phone-icon.png" alt="flowerShopLogo">
+        </div>
+      
+
+        <div class="col-sm-3 mt-5">
+          <p>text goes here</p>
+        </div>
+      </div>
+      <div class="d-flex justify-content-center"> 
+        <div class="col-sm-3">
+          <img src="./assets/images/truck-icon.png" alt="flowerShopLogo">
 
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
+          </div>
+        
 
-    <!-- Delivery Div -->
+          <div class="col-sm-3 mt-5">
+            <p>text goes here</p>
+          </div>
+        
+      </div> -->
 
     <!--Scripts required for bootstrap CDN-->
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
   </body>
 </html>

```

Comment:
With this exercise, there is an issue with the background image. You could have used CSS to add in the background -image of the flower shop and then the text and button using the Z-Index.